src/main/java/life/qbic/portal/utils/PropertiesBasedConfigurationManager.java

```diff
@@ -258,6 +267,17 @@
     }
   }
 
+  private void createServiceUserFrom(Properties properties) {
```

Comment:
```suggestion
  private void createServiceUser(Properties properties) {
```
```suggestion
  private void createServiceUserFromProperties(Properties properties) {
```
I get what you meant to do here. I would suggest to either leave it out or include it in the method name rather than the parameter name. (since those are not necessarily available from any IDE and it is confusing to only see `createServiceUserFrom`)